L1Trigger/RPCTechnicalTrigger/interface/TTUTrackingAlg.h

```diff
@@ -75,25 +66,22 @@ class TTUTrackingAlg : public TTULogic {
   public:
     
     Track() { m_tracklength = 0; };
-    ~Track() { 
-      if ( m_tracklength < 0 ) delete m_seeds[0];
-      m_seeds.clear();
-    };
-    
-    Track( const Track & trk ) 
-    {
-      m_seeds = trk.m_seeds;
-      m_tracklength = trk.m_tracklength;
-    };
-    
+
+    Track( const Track&) = delete;
```

Comment:
Because it holds a pointer, pointing into memory it controls, non of these functions will work correctly.